.github/workflows/tag-more-reviewers.yaml

```diff
@@ -1,7 +1,7 @@
 name: Tag more reviewers
 
 on:
-  pull_request:
+  pull_request_target:
```

Comment:
what is the reason for this change?

Reply:
There's also https://securitylab.github.com/resources/github-actions-preventing-pwn-requests/ which confuses more than it explains anything.

My impression that `pull_request_target` is helpful when we need to do something on PRs from **forks**. `pull_request` should work just as well on PRs in the same repo. Since MintMaker PRs aren't on forks, I think we should change that. WDYT?

Reply:
BTW, this question with the top reply are helpful https://stackoverflow.com/questions/74957218/what-is-the-difference-between-pull-request-and-pull-request-target-event-in-git